### DISCUSSION TITLE
Добавьте ссылку-приглашение на сервер

### DISCUSSION THREAD

**Discussion by dartanyana** [stoatchat/discussions#805] [Feature Suggestions]
Привет! Добавьте, пожалуйста, возможность пригласить на сервер по ссылке, хочу позвать коллег, чтобы решать рабочие вопросы на вашей площадке. Спасибо вам за ваши труды :)
![Скриншот 09 10 24_02 29 44](https://github.com/user-attachments/assets/c48ca235-36d2-4a64-847e-7bcee7cd56e8)
![Скриншот 09 10 24_02 29 35](https://github.com/user-attachments/assets/f9789648-ea11-426f-a6a3-2ad818e8b758)

**Comment by stacesrk** [comment]
Вы уже можете сделать это, щелкнув правой кнопкой мыши на любой канал на сервере, и там будет "создать приглашение", далее скопировать ссылку
